docs/operator-manual/troubleshooting.md

```diff
@@ -0,0 +1,55 @@
+# Troubleshooting Tools
+
+The document describes how to use `argocd-tool` binary to simplify Argo CD settings customizations and troubleshot
+connectivity issues.
+
+## Settings
+
+Argo CD provides multiple ways to customize system behavior and has a lot of settings. It might be dangerous to modify
+settings on Argo CD used in production by multiple users. Before applying settings you can use `argocd-util` binary to
+make sure that settings are valid and Argo CD is working as expected. The `argocd-util` binary is available in `argocd`
+mage and might be used using docker. Example:
+
+```bash
+docker run --rm -it -w /src -v $(pwd):/src argoproj/argocd:<version> \
+  argocd-util settings validate --argocd-cm-path ./argocd-cm.yaml
+```
+
+The `argocd argocd-util settings validate` command performs basic settings validation and print short summary
```

Comment:
I think it is just `argocd-util settings validate` instead of  `argo argocd-util settings validate`